### PR TITLE
github: update of .jira_sync_config.yaml

### PR DESCRIPTION
**Pull request by mseralessandri**
I  changed the 'jira_key' and 'components' in .jira_sync_config.yaml configuration file for the synchronization with Jira.
The issues in MicroCluster will be created in a separated epic and the 'component' key will allow filtering by 'MicroCluster' key

### CODE REVIEW

File: .github/.jira_sync_config.yaml [hunk 1]
```diff
@@ -28,12 +27,12 @@ settings:
   sync_comments: false
   
   # (Optional) (Default: None) Parent Epic key to link the issue to
-  epic_key: "LXD-1251"
+  epic_key: "LXD-2467"
```
Comment (roosterfish) [inline .github/.jira_sync_config.yaml:30]: Will the previously synced stories be moved into the new epic?

Comment (mseralessandri) [reply .github/.jira_sync_config.yaml:30]: Unfortunately not, I need to change the parent in Jira. The synchronization however works even if you move the issues in another epic.

Comment (roosterfish) [reply .github/.jira_sync_config.yaml:30]: Ok thanks!

**Review by roosterfish — APPROVED** [review]

**Comment by roosterfish** [comment]
@mseralessandri can all of those bug sync epics be linked under a central epic (or another type) so they can be found easily in Jira? Just an idea.

**Comment by mseralessandri** [comment]
> @mseralessandri can all of those bug sync epics be linked under a central epic (or another type) so they can be found easily in Jira? Just an idea.

You can link the epics only to objectives but then they would appear in the road map

**Pull request merged**